### PR TITLE
Update to Rails 5.2

### PR DESCRIPTION
**Pull request by vipulnsward**

### CODE REVIEW

**Comment by neerajdotname** [comment]
Verified it on heroku https://www.evernote.com/l/AVZ1zItipJVPK5-MoHKBWr4cuvPBgaRoAtc

**Comment by neerajdotname** [comment]
@vipulnsward LGTM. Go ahead and merge it if you think upgrade work is done.

**Comment by vipulnsward** [comment]
I want to play around with secrets and move it to encrypted credentials.

Will merge tomorrow.

File: Gemfile [hunk 1]
```diff
@@ -108,7 +111,7 @@ group :development do
   gem "web-console", "~> 3.0"
 
   # Reenable after https://github.com/rails/rails/issues/26158 is fixed
-  # gem 'listen', '~> 3.0.5'
```
Comment (vipulnsward) [inline Gemfile]: http://guides.rubyonrails.org/security.html#custom-credentials doesnt work without this gem

**Comment by neerajdotname** [comment]
Closing in favor of https://github.com/bigbinary/wheel/pull/193

**Pull request closed without merging**